src/components/Secrets.vue

```diff
@@ -34,7 +40,6 @@ export default {
 
 <style scoped>
 img{
-  width: 50px;
```

Comment:
I think I would leave this, it makes some images look a bit strange, but they all have the same dimensions, which looks cleaner in my opinion.